packages/firestore/test/unit/api/timestamp.test.ts

```diff
@@ -16,12 +16,55 @@
  */
 
 import { expect } from 'chai';
+import { Code } from '../../../src/util/error';
 import { Timestamp } from '../../../src/api/timestamp';
 import { addEqualityMatcher } from '../../util/equality_matcher';
 
 describe('Timestamp', () => {
   addEqualityMatcher();
 
+  it('constructor should validate the "seconds" argument and store it.', () => {
```

Comment:
Nit: I think in general we don't use period in our test names.

Reply:
Fixed.